src/objects.js

```diff
@@ -1,41 +1,68 @@
 const createPerson = (name, age) => {
   // your code here
+  const person = new Object();
+  person.name = name, 
+  person.age = age;
+  
+  return person;
```

Comment:
Rather than creating anew object like this and assigning properties, you could instead just return an object directly like this:
```
  return {
    name: name,
    age: age
  };
```
There is a new shorthand way of writing this too, when the keys and values are the same you can just write them singly like this:
```
  return {
    name,
    age
  };
```